templates/make_manifest

```diff
@@ -7,18 +7,18 @@ STEMCELL_OS=${STEMCELL_OS:-ubuntu}
 
 infrastructure=$1
 servers=$2
-shift
-shift
 
-if [ "$infrastructure" != "aws-ec2" ] && \
-    [ "$infrastructure" != "openstack" ] && \
-    [ "$infrastructure" != "warden" ] || \
-    [ -z "$servers" ]; then
-  echo $infrastructure
-  echo "usage: ./make_manifest <aws-ec2|warden|openstack> <backend_server1>,[<backend_server2>,...,<backend_serverN>]"
+if [[ "$infrastructure" != "aws-ec2" ]] && \
+    [[ "$infrastructure" != "openstack" ]] && \
+    [[ "$infrastructure" != "warden" ]] || \
+    [[ -z "$servers" ]]; then
+  echo >&2 "usage: ./make_manifest <aws-ec2|warden|openstack> <backend_server1>,[<backend_server2>,...,<backend_serverN>]"
```

Comment:
Does the script even use `$infrastructure` any more, given the use of cloud-config?